libraries/AP_BattMonitor/AP_BattMonitor_ADS1115.cpp

```diff
@@ -29,16 +29,16 @@ AP_BattMonitor_ADS1115::read()
 	_adc.read(report, 4);
 
     // get voltage
-    _state.voltage = report[2].data;
+    _state.voltage = report[3].data * 0.01;
 
 	// calculate time since last current read
 	uint32_t tnow = AP_HAL::micros();
 	float dt = tnow - _state.last_time_micros;
     // get amperage
-    _state.current_amps = report[3].data;
+    _state.current_amps = report[2].data * -1;
     
     #if BATTMON_ADS1115_DEBUG
-    hal.console->printf("%2.8F volts, %2.8F amps", _state.voltage, _state.current_amps);
+    hal.console->printf("%2.8F volts, %2.8F mamps \n", _state.voltage, _state.current_amps);
```

Comment:
It'll probably be easier to read the debug printout if there's just one new line per cycle. Try removing the new line here, but add a command and another trailing space, e.g.
```
printf("%2.8F volts, %2.8F mamps, ", ...);
```
Beyond that, there's probably no reason to print out up to 8 decimal points - it's really just cluttering the console printout with sig figs that don't really mean anything to the reader. I would at most print out 2 decimal points, e.g. "%.2f".